streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamImpl.java

```diff
@@ -792,16 +803,13 @@ public void process(final ProcessorSupplier<? super K, ? super V> processorSuppl
         selectKeyMapNode.keyChangingOperation(true);
 
         builder.addGraphNode(this.streamsGraphNode, selectKeyMapNode);
-        return new KGroupedStreamImpl<>(
-            builder,
-            selectKeyMapNode.nodeName(),
-            sourceNodes,
-            serializedInternal.keySerde(),
-            serializedInternal.valueSerde(),
-            true,
-            selectKeyMapNode
-        );
-
+        return new KGroupedStreamImpl<>(selectKeyMapNode.nodeName(),
```

Comment:
super nit: why the change in the order of constructor args?

Reply:
My personal paranoid-taste: to keep the parameters with similar length together :P